tensorbay/client/dataset.py

```diff
@@ -464,16 +464,16 @@ def commit(self, message: str, *, tag: Optional[str] = None) -> None:
         self._status.check_authority_for_draft()
         self._status.checkout(commit_id=self._commit(message, tag))
 
-    def update_notes(self, *, is_continuous: bool) -> None:
+    def update_notes(self, notes: Notes) -> None:
```

Comment:
updateNote OpenAPI use `PATCH` method to support change specific field in `notes`
if our `update_notes` function use `Notes` instance as input, how can we support to change specific field?